docker-compose.example.sqlite.yml

```diff
@@ -1,20 +1,26 @@
-version: "3.7"
+version: "2.4"
 services:
   app:
     image: babybuddy/babybuddy
+    # Container_name is an optional value for single-container deployments to set a friendly container name.
+    # Disable in swarm or other multi-container setups.
+    container_name: babybuddy
     # See README.md#configuration for other environment configuration options.
     environment:
-      - ALLOWED_HOSTS=
+      - ALLOWED_HOSTS=localhost   # comma separated list of IP addresses or hosts that can access the web UI
```

Comment:
Does `localhost` make this "just work" with compose? If so this seems like a good change.

Reply:
I haven't tested all the permutations of this, but at least for someone running it on their own machine "localhost" would "just work". Most other users will have to add an additional comma separated host. The debug mode in Django seems to allow for [*], but that seemed like an unwise default for production.